src/CoreWCF.Primitives/src/CoreWCF/Security/SecurityProtocol.cs

```diff
@@ -620,8 +620,13 @@ protected virtual void OnVerifyIncomingMessageFailure(Message message, Exception
             return authenticators;
         }
 
+        public Task CloseAsync(TimeSpan timeout)
+        {
+            return this.CommunicationObject.CloseAsync();
+        }
```

Comment:
Missing space below. Also, this isn't what the original method looked like. This is what the original looked like:
```c#
        public void Close(bool aborted, TimeSpan timeout)
        {
            if (aborted)
            {
                this.communicationObject.Abort();
            }
            else
            {
                this.communicationObject.Close(timeout);
            }
        }
```
The abort code path is different.

Reply:
made it same.. the abort in this case doesn't do anything so I didn't see any error.. but you are right... let's try to make it as exact as possible..